src/screens/monitor/components/documentTable/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { GenericTable } from "components/genericTable"
+import { GenericTableRow } from 'components/genericTableRow';
+import { DocumentDto } from 'services/api/dtos';
+
+interface DocumentsTableProps {
+  documents: DocumentDto[];
+};
+
+export const DocumentsTable = (props: DocumentsTableProps) => {
+  const { documents } = props;
+
+  const columns = [
+    'Identificator',
+    'Titlu',
+    'Proiect',
+    'Data publicarii',
+    'Sursa',
+    'Stare',
+    'Termeni identificati'
+  ]
+
+  const documentRows = documents && documents.map(document => (
```

Comment:
Documents check should be moved outside of the component. In the eventuality of no data we might use some kind of message to notify the user. The message should not be the responsibility of the DocumentsTable